database/factories/QuizFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Quiz>
+ */
+class QuizFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition()
+    {
+        return [
+            'title' => ['en' => 'quiz'],
+            'user_id' => User::factory(),
```

Comment:
Should be removed. But maybe needs course or module relationships?

Reply:
I think not assigning course or module id to quiz factory makes sense, because they are optional and the test cases I wrote creates quizzes for either course or module.